works/filters.py

```diff
@@ -0,0 +1,57 @@
+import django_filters
+from . import models as works_models
+
+
+class WorkTypeFilter(django_filters.rest_framework.FilterSet):
```

Comment:
Please add a few comments here. I understand what you did after reading [filtering](http://www.django-rest-framework.org/api-guide/filtering/). But it might not be so obvious.